packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js

```diff
@@ -185,7 +185,7 @@ storiesOf("Molecules|Select", module)
         :disabled="disabled"
         >
         <SfSelectOption v-for="(option, key) in options" :key="key" :value="option.value">
-          <SfProductOption :color="option.color" :label="option.label"></SfProductOption>
+          <span :color="option.color" :label="option.label">{{option.value}</span>
```

Comment:
missing second }

Reply:
I fixed that too and removed span as it is not the best idea to put additional element inside <option>. It could also give ssr mismatch error.